hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/SparkRDDWriteClient.java

```diff
@@ -200,7 +200,8 @@ public HoodieWriteResult insertOverwrite(JavaRDD<HoodieRecord<T>> records, final
     table.validateInsertSchema();
     preWrite(instantTime, WriteOperationType.INSERT_OVERWRITE, table.getMetaClient());
     HoodieWriteMetadata result = table.insertOverwrite(context, instantTime, records);
-    return new HoodieWriteResult(postWrite(result, instantTime, table), result.getPartitionToReplaceFileIds());
+    return new HoodieWriteResult(postWrite(result, instantTime, table),
```

Comment:
Would be good to avoiid unrelated cosmetic changes and do them separately :)

Reply:
I used the plugin to adhere to code style, which might have caused it. I will change it back to the previous state